MMUL-BASH-01/Exercises/Day-4/Exercise.006.solutions

```diff
@@ -0,0 +1,42 @@
+# Exercise 006 - Print every element of the array
```

Comment:
Still missing " - Solutions", still repeating questions instead of writing answers.